camel-core/src/main/java/org/apache/camel/util/jsse/SSLContextClientParameters.java

```diff
@@ -34,6 +37,19 @@
     
     private static final Logger LOG = LoggerFactory.getLogger(SSLContextClientParameters.class);
 
+    private List<SNIServerName> sniHostNames = new ArrayList<>();
+
+    public void setSniHostNames(List<String> sniHostNames) {
```

Comment:
Maybe let it be a plain getter/setter. As this setter will add and not remove any existing. A bit unusual, as getter/setter are usually just to set or get the values as-is.